module_9/test_locators.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+from typing import TYPE_CHECKING
+import locators as l
+from utils import *
```

Comment:
It is better to avoid such import, and import a specific function or a list of functions instead